procedures/misc-operations/puppet-pyup.rst

```diff
@@ -195,6 +195,6 @@ The list of people who can ssh in is `here
 ($shortlist) <https://github.com/mozilla-releng/build-puppet/blob/master/manifests/moco-config.pp#L196-L212>`__
 
 Success
-=======
+-------
```

Comment:
This is to remove this section from the [index](http://docs.mozilla-releng.net/en/latest/procedures/index.html)